### DISCUSSION TITLE
An option to remove Explorer context menu integration

### DISCUSSION THREAD

**Discussion by b00rya** [GSDragoon/RadeonSoftwareSlimmer#20] [Ideas]
same as powershell command:
Get-AppxPackage -AllUsers AdvancedMicroDevicesInc-RSXCM | Remove-AppxPackage -AllUsers

**Comment by GSDragoon** [comment]
That app is also responsible for a startup task to load Radeon Software on logon/reboot. Is there a way to disable the context menu item without removing the app completely?

**Reply by b00rya** [reply]
I don't know if there is a way, there is little information about it =(
Seems like it's crafted specifically to make it hard to disable
